packages/console/src/nodes/Select.tsx

```diff
@@ -0,0 +1,172 @@
+import type { ReactElement } from "react";
+import { useState } from "react";
+
+import { Select as MuiSelect } from "@mui/material";
+import FormControl from "@mui/material/FormControl";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
```

Comment:
Merge all the import statements. Instead of importing each component per import statement, you can import it like:
```
import { Select as MuiSelect, FormControl, InputLabel, MenuItem } from "@mui/material";
```